src/index.ts

```diff
@@ -1,7 +1,12 @@
 import app from './app';
+import dbConnect from './dbConnect'
 
 const PORT = process.env.PORT || 3000;
 
+(async () => {
+    await dbConnect();
+})();
```

Comment:
I don't think this will block the execution of `app.listen`. If you want to wait for the connection you'll need to call `listen` inside a `then` tacked on to `dbConnect()` or write an async startup function that does `await dbConnect()` the calls `app.listen`